install.sh

```diff
@@ -2148,6 +1580,39 @@ DATASTORE_DRIVER_ISCSI_SCRIPTS="src/datastore_mad/remotes/iscsi_libvirt/cp \
                          src/datastore_mad/remotes/iscsi_libvirt/snap_flatten \
                          src/datastore_mad/remotes/iscsi_libvirt/clone"
 
+DATASTORE_DRIVER_RSYNC_SCRIPTS="src/datastore_mad/remotes/rsync/cp \
+                         src/datastore_mad/remotes/rsync/mkfs \
+                         src/datastore_mad/remotes/rsync/stat \
+                         src/datastore_mad/remotes/rsync/clone \
+                         src/datastore_mad/remotes/rsync/monitor \
+                         src/datastore_mad/remotes/rsync/snap_delete \
+                         src/datastore_mad/remotes/rsync/snap_revert \
+                         src/datastore_mad/remotes/rsync/snap_flatten \
+                         src/datastore_mad/remotes/rsync/rm \
+                         src/datastore_mad/remotes/rsync/backup \
+                         src/datastore_mad/remotes/rsync/backup_cancel \
+                         src/datastore_mad/remotes/rsync/restore \
+                         src/datastore_mad/remotes/rsync/export \
+                         src/datastore_mad/remotes/rsync/increment_flatten \
+                         src/datastore_mad/remotes/rsync/ls"
+
+DATASTORE_DRIVER_RESTIC_SCRIPTS="src/datastore_mad/remotes/restic/cp \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

DATASTORE_DRIVER_RESTIC_SCRIPTS appears unused. Verify use (or export if used externally).